.storybook/preview.js

```diff
@@ -11,6 +11,7 @@ import {Formio, Templates} from 'react-formio';
 import {fixIconUrls as fixLeafletIconUrls} from 'map';
 import 'scss/dte-theme.scss';
 import 'leaflet/dist/leaflet.css';
+import 'leaflet-geosearch/dist/geosearch.css';
```

Comment:
this should not be needed anymore because of https://github.com/open-formulieren/open-forms-sdk/pull/486/files#diff-8b021783ff171ffef1c5228dfc0ff05cc75ba3a05883fc133f76cf25fda22d03R2